bylaws/003-membership.md

```diff
@@ -128,5 +128,121 @@ available.
+other duties as required. As the role has continuity (in that there will always
+be a post holder) and redundancy (There are a number of post holders in case of
+the absence of one) it can be ensured that responsibilities assigned to the
+secretary will always be completed and therefore administrative responsibilities
+such as vote management should always be assigned to the FIG secretary.
+
+All FIG secretaries are expected to remain impartial and professional when acting
+in an official capacity as secretary and should also remain aware that even when
+not acting in an official capacity, their actions reflect back on the PHP FIG and
+the FIG must not be brought into disrepute.
+
+### Defined functions
+
+There are a number of defined functions that the FIG secretaries are expected to
+complete but they may also, within their remit defined in the above overaching
+role above, perform other duties as nessasary.
+
+* Keeping the website, twitter and other marketing mediums up-to-date
+* GitHub organization administration
+* Tallying votes
+* Tracking member project activity, and marking projects as inactive
+* Ensuring bylaws are being followed
+* Clarifying any interpretation of bylaw text (If there is lack of consensus
+between FIG secretaries then it must be put to a full vote according to the
+standard [Voting Protocol bylaw][voting]).
+* Ensure that relevant marketing mediums (e.g. Twitter and Facebook) are kept
+professional, up to date and impartial.
+* Moderate discussions on github, the mailing list and IRC channels to ensure
+that an appropriate environment is maintained
+* Should there be a FIG meeting at a conference or other ad hoc gathering any
+Secretary in attendance should take notes to report back to the mailing list.
+
+### Access and Abilities
+
+FIG secretaries will be given access to the GitHub organisation as 'Owners' and
+full (admin) access to anything relating to official website, marketing and
+communication mediums including but not limited to the domain, twitter, IRC
+channel, packagist packages and mailing list in order to allow them to complete
+their duties and ensure that FIG mediums are managed by representatives elected
+by the PHP FIG.
+
+Also, whilst not full members themseleves, they have the ability to start any
+vote (just as any member project), but not cast any votes. However it should be
+noted that any vote started by one FIG secretary must be managed and tallied by a
+different secretary to ensure impartiality.
+
+### Selection:
+
+Secretaries are selected every year at the end of January after a vote according
+to standard [Voting Protocol bylaw][voting] except with the change that any
+outgoing secretaries may also participate in the vote. Secretaries are then in
+post until such a time as they resign, are removed by a vote of no confidence or
+until the following January.
+
+At all times there should be at one or more FIG secretary from the previous term
+standing for re-election.
```

Comment:
`at least one` or `one or more`  would be correct but not both. But thanks for pointing this out.